src/vmq_cloudwatch_metrics.erl

```diff
@@ -260,3 +271,25 @@ unit({_Type, _Metric}) ->
 value(V) when is_integer(V) -> float(V);
 value(V) when is_float(V)   -> V;
 value(_) -> 0.0.
+
+%%--------------------------------------------------------------------
+%% @private
+%% @doc
+%% Determines whether or not AWS Credentials are provided in the
+%% VerneMQ config.
+%% @end
+%%--------------------------------------------------------------------
+-spec has_config_credentials(iodata(), iodata()) -> boolean().
+has_config_credentials(undefined, _) ->
+    false;
+has_config_credentials(_, undefined) ->
+    false;
+has_config_credentials(undefined, undefined) ->
+    false;
+has_config_credentials(AccessKeyID, SecretAccessKey) when
+is_list(AccessKeyID), is_list(SecretAccessKey) ->
```

Comment:
I assume that this is correct, but it _seems_ wrong to me that we're checking if it's a list when I would expect it to be a string.  What am I misunderstanding about this?

Reply:
I believe strings are lists in erlang?

Reply:
I thought that might be a possibility, but seemed strange.  I think you're correct though: `a list of codepoints`

Reply:
In Erlang Starings are lists, otherwise, they could be binaries